tui/offers_list.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+from pyppeteer.element_handle import ElementHandle
+from pyppeteer.page import Page
+from pyppeteer.browser import Browser
+from tqdm import tqdm
+
+from common import accept_cookies
+
+
+async def _get_all_offers_elements(page: Page):
+    results = await page.querySelectorAll(".results-container")
+    assert len(results) == 1
+    return await results[0].querySelectorAll(".offer-tile-wrapper")
+
+
+async def _get_offer_link(offer: ElementHandle) -> str:
+    offer_a = await offer.querySelector("a")
+    assert offer_a is not None, "Oferta nie ma linku"
+    href = await offer_a.getProperty("href")
+
+    return str(await href.jsonValue())
+
+
+async def get_all_offers(browser: Browser,
```

Comment:
```suggestion
async def get_all_offers(
    browser: Browser,
    url: str,
    iterations: int = 1
):
```
brakuje tutaj też return type'u